package/MDAnalysis/analysis/align.py

```diff
@@ -373,6 +373,13 @@ def alignto(mobile, reference, select=None, weights=None,
     `reference`, therefore preserving the exact atom ordering (see
     :ref:`ordered-selections-label`).
 
+    .. Warning:: 2.2.0
+       In version 3.0.0, when the reference and mobile selections are atom
+       groups from the same universe and ``subselection`` is ``None``,
+       the reference selection will no longer be transformed. This will
+       result in the same behaviour what is currently obtained in the above
+       case but with ``subselection`` set to 'all'.
```

Comment:
This reads as if the issue is the future change to 3.0.0.

What you need to do instead is state the current problem, how it's affecting users and say that it will be changed in version 3.0.0 and how.

Please reword this accordingly.